Article/Article.js

```diff
@@ -2,19 +2,20 @@
 
 class Article {
   constructor(domElement) {
+
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'Read More';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
```

Comment:
You could do this.expandButton.addEventListener('click',() => { this.expandArticle() })